Jenkinsfile-deploy

```diff
@@ -5,7 +5,7 @@ pipeline {
     stage('Test') {
         agent {
             ecs {
-                inheritFrom 'ecs'
+                inheritFrom 'sbt'
```

Comment:
Does the other Jenkinsfile need this line as well?

Out of curiosity, where is this template defined?

Reply:
I've changed it in the other file. They're defined in the Jenkins project here https://github.com/nationalarchives/tdr-jenkins/tree/master/docker. I've started with one folder per template.